pyhuman/human.py

```diff
@@ -84,9 +95,11 @@ def signal_handler(sig, frame):
     parser.add_argument('--taskgroupinterval', type=int, default=GROUPING_INTERVAL_SECONDS)
     parser.add_argument('--stopafter', type=int, default=HUMAN_LIFESPAN_SECONDS)
     parser.add_argument('--extra', nargs='*', default=EXTRA_DEFAULTS)
+    parser.add_argument('--seed', type=int, default=0)
```

Comment:
I don't like this diff.  The default previously was to use the `current system time` according to https://www.w3schools.com/python/ref_random_seed.asp.  Seems like we should keep that unless the user declares a seed be used.

Reply:
ok i put back original behavior